packages/plugins/rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -91,6 +91,9 @@ gem build %{gem_name}.gemspec
 
 %pre
 if [ -d %{foreman_proxy_dir}/.ssh ] && [ ! -L %{foreman_proxy_dir}/.ssh ] ; then
+  if [ -d %{foreman_proxy_statedir}/ssh ] ; then
+    mv %{foreman_proxy_statedir}/ssh %{foreman_proxy_statedir}/ssh.save$(date '+%%y%%m%%d%%H%%M')
```

Comment:
Ideally this will run only once, so technically adding the date to the backup-directory should not be necessary.
I hope I did escape the `%` in the `date`-command correctly :crossed_fingers: